labs/computing-experiment-metrics/index.md

```diff
@@ -13,13 +13,13 @@ In order to measure the impact this banner has on customer support volumes and d
 <table>
     <tr>
         <td>
-            <img src="https://raw.githubusercontent.com/optimizely/labs/master/labs/computing-experiment-subjects/img/control.png" alt="Control" style="width:100%; padding-left:0px">
+            <img src="img/control.png" alt="Control" style="width:100%; padding-left:0px">
```

Comment:
These relative URLs currently won't work when it's published on optimizely.com/labs